### ISSUE TITLE
Unable to manage locks on Samsung Connect

### ISSUE THREAD

**Issue by danschlim** [ethayer/lock-manager#105]
I have had this app for a while and it works great for managing my Schlage locks on the Smartthings (Classic) app.  I have noticed that once I try to use the new Samsung Smartthings app (Formerly, Samsung Connect) I am not able to open the locks from the device list nor am I able to open the Lock Manager smart app.

Has anyone experienced this issue?  if so, is there a fix?

**Comment by grainsoflight** [comment]
The Samsung Connect app is atrocious. Samsung basically recommends you use the old smart things app unless absolutely necessary, as they are still in the process of migrating the old functionality to the new app. Problem is they've "Been in the process," for months. Samsung basically released an app knowing full well it doesn't work.

TL;DR: Use the old app.
